lib/scrapers/twitter.js

```diff
@@ -50,7 +50,7 @@
         return (function(__iced_k) {
           __iced_deferrals = new iced.Deferrals(__iced_k, {
             parent: ___iced_passed_deferral,
-            filename: "/home/me/code/proofs/src/scrapers/twitter.iced",
+            filename: "/Users/miles/go/src/github.com/keybase/proofs/src/scrapers/twitter.iced",
```

Comment:
do you think you can do search and replace for these to make diff smaller?

Reply:
I think it's unfortunate that the build system is not reproducible but why bother making a diff smaller?